azure-simple/terraform/variables.tf

```diff
@@ -0,0 +1,75 @@
+# Define variables.
```

Comment:
Is it possible to define the License header for Terraform files? if so we may need to add that.

Reply:
Added the License header.